pages/common/drupal-check.md

```diff
@@ -0,0 +1,20 @@
+# drupal-check
+
+> Check Drupal PHP code for deprecations.
+> More information: <https://github.com/mglaman/drupal-check>.
+
+- Check the code in a specific path for deprecations:
+
+`drupal-check {{path/to/directory}}`
+
+- Exclude a directory:
+
+`drupal-check --exclude-dir {{path/to/directory/to/exclude}} {{path/to/directory}}`
```

Comment:
:thinking:  if the man page says:
```
-e Exclude directories. Wildcards work. Separate multiple excluded directories with commas, no spaces. e.g.: */tests/codeception/acceptance/*.php
```
Maybe we can add this into the example:
```suggestion
`drupal-check --exclude-dir {{path/to/excluded_directory}},{{path/to/excluded_files/*.php}} {{path/to/directory}}`
```
or just
```suggestion
`drupal-check --exclude-dir {{path/to/excluded_directory2}},{{path/to/excluded_directory2}} {{path/to/directory}}`
```

Reply:
I'll admit I don't use this option but I agree it would useful to mention it.